### PR TITLE
Edge. CIE-2833. Updated instructions for remote and local diagnostics

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: Downloading diagnostics using the UI 
+layout: redirect
+---
+
+1. Log in to the {{< management-tenant >}} using the Edge administrator credentials created during the installation.
+
+	- Username: management/<*Edge admin username*>
+	- Password: password provided during the installation
+
+2. Switch to the **Administration** application using the application switcher at the right of the top bar **<img class="Default" src="/images/icons/switcher-icon.png" alt="icon" style="display: inline; float: none">**.
```
Comment (BeateRixen) [inline content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md:12]: **Administration** application > Administration application

File: content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+3. Click **Edge** > **Diagnostics** in the navigator.
+
+4. Click **Request diagnostics report** to download the diagnostics.
+
+To download the diagnostics report remotely, see [Downloading diagnostics report remotely](/edge/remote-connectivity/#downloading-diagnostics-remotely).
```
Comment (BeateRixen) [inline content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md:18]: [Downloading diagnostics report remotely] > [Remote connectivity > Downloading diagnostics report remotely]

I realized that the title of the chapter should not be spelled with a hyphen: "Remote-connectivity" should be Remote connectivity".

File: content/edge/remote-connectivity-bundle/downloading-diagnostics-remotely.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+weight: 46
+title: Downloading diagnostics remotely
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/downloading-diagnostics-remotely.md:3]: The link above says "Download diagnostics report remotely", so change either the one or the other.

File: content/edge/remote-connectivity-bundle/downloading-diagnostics-remotely.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+layout: redirect
+---
+
+The Cumulocity IoT tenant allows you to download the diagnostics report remotely from your tenant account after you have registered your Edge appliance in the tenant account.
```
Comment (BeateRixen) [inline content/edge/remote-connectivity-bundle/downloading-diagnostics-remotely.md:7]: Cumulocity IoT > use variable

File: content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md (hunk 1 above)
Comment (sharath-jk) [reply content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md:12]: Done.

File: content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/diagnostics-and-support-bundle/diagnostic-report-through-ui.md:18]: Done.

File: content/edge/remote-connectivity-bundle/downloading-diagnostics-remotely.md (hunk 4 above)
Comment (sharath-jk) [reply content/edge/remote-connectivity-bundle/downloading-diagnostics-remotely.md:7]: Done.

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**